srcpkgs/corkscrew/template

```diff
@@ -1,12 +1,16 @@
 # Template file for 'corkscrew'
 pkgname=corkscrew
 version=2.0
-revision=1
+revision=2
 build_style=gnu-configure
+hostmakedepends="automake"
 short_desc="Corkscrew is a tool for tunneling SSH through HTTP proxies"
 maintainer="pulux <pulux@pf4sh.de>"
-license="GPL-2"
-homepage="http://www.agroman.net/corkscrew/"
-#distfiles="http://agroman.net/corkscrew/$pkgname-$version.tar.gz"
-distfiles="https://ftp.heanet.ie/mirrors/OpenBSD/distfiles/corkscrew-${version}.tar.gz"
-checksum=0d0fcbb41cba4a81c4ab494459472086f377f9edb78a2e2238ed19b58956b0be
+license="GPL-2.0-or-later"
+homepage="https://github.com/bryanpkc/corkscrew"
+distfiles="https://github.com/bryanpkc/corkscrew/archive/refs/tags/v${version}.tar.gz"
+checksum=09d70daaa3145070a1e463dbab6ae1cbc62449b1e09c94e158f6017e7c79c9f6
+
+pre_configure() {
+	autoreconf -fi
+}
```

Comment:
Why do we need autoreconf?

Reply:
Because without autoreconf the configure script isn't available and it dies.